### PR TITLE
Fix and standardize styling for all tables

### PR DESCRIPTION
**Pull request by phuongdh**
# Before
Screenshot from 2020-01-23 16-14-35

# After
Screenshot from 2020-01-23 16-14-49

Screenshot from 2020-01-23 16-16-44

### CODE REVIEW

**Comment by mcwhitaker** [comment]
Looks great!!!!

File: config/settings.yml [hunk 1]
```diff
@@ -62,6 +62,8 @@ bib_retriever:
     retriever_class_require: 'avalon/bib_retriever/sru'
 controlled_vocabulary:
   path: config/controlled_vocabulary.yml
+timeliner:
+  timeliner_url: http://localhost:3000/timeliner
```
Comment (Dananji) [inline config/settings.yml]: @phuongdh Is it okay to push settings changes?

Comment (phuongdh) [reply config/settings.yml]: Hmm now that I think about it, it should probably live in `config/settings/development.yml` instead

Comment (phuongdh) [reply config/settings.yml]: Split out to a different PR

**Pull request merged**